### PR TITLE
Removed outdated references to contact us.

### PR DESCRIPTION
**Pull request by drazisil**

### CODE REVIEW

File: jekyll/_docs/build-image-precise.md [hunk 1]
```diff
@@ -8,10 +8,6 @@ changefreq: "weekly"
 
 Occasionally, bugs in tests arise because CircleCI's environment differs slightly from your local environment.
 In the future, we plan to allow as much of this to be configurable as possible.
-Please [contact us](mailto:support@circleci.com) if some part of our environment is not suitable for you, and we will try to come up with a workaround.
-
-If any version is not listed here, SSH into our build boxes to check it manually (and [contact us](mailto:support@circleci.com)
```
Comment (li3n3) [inline jekyll/_docs/build-image-precise.md]: Is this also something that's irrelevant at this point? (or maybe it never was)

File: jekyll/_docs/configuration.md [hunk 2]
```diff
@@ -15,10 +15,6 @@ file, check out our [sample file]({{site.baseurl}}/config-sample/).
 
 Should you have a test failure, our [troubleshooting section]({{ site.baseurl }}/troubleshooting/)
 can likely tell you the best way to solve the problem.
-If you find yourself repeatedly consulting this guide, please
-[contact us](mailto:support@circleci.com) and let us know what you're working on.
-We'll try to make it easier for you.
```
Comment (li3n3) [inline jekyll/_docs/configuration.md]: Do you think this is any different than the other ones? It strikes me as a bit more general, more a request for "let us know if there are rough spots," but I'm totally okay with axing it if it is also outdated.

File: jekyll/_docs/build-image-precise.md (hunk 1 above)
Comment (drazisil) [inline jekyll/_docs/build-image-precise.md]: As feature requests and newer versions are handled by discuss our docs should not ask customers to email us for these things.

File: jekyll/_docs/configuration.md (hunk 2 above)
Comment (drazisil) [inline jekyll/_docs/configuration.md]: I feel document improvement requests should be via PR, not email as this repo is public.

**Pull request merged**